client/landing/stepper/declarative-flow/internals/steps-repository/ecommerce-segmentation-survey/queries/use-segmentation-survey-query.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query';
+import { Survey } from 'calypso/components/survey-container/types';
+import wpcom from 'calypso/lib/wp';
+
+const useSegmentationSurveyQuery = () => {
+	return useQuery< Survey >( {
+		queryKey: [ 'segmentation-survey' ],
+		queryFn: () => {
+			return wpcom.req.get( {
+				path: `/segmentation-survey`,
```

Comment:
`survey_key` will be mandatory in this query.

Reply:
This query is not retrieving the answers but the survey structure. Is that what you refer to? @JuanLucha